rally/track.json

```diff
@@ -0,0 +1,55 @@
+{% import "rally.helpers" as rally with context %}
+{
+  "version": 2,
+  "description": "APM benchmark for Rally",
+  "indices": [
+    {
+      "name": "apm-error",
+      "body": "index.json",
+      "types": [ "type" ]
+    },
+    {
+      "name": "apm-transaction",
+      "body": "index.json",
+      "types": [ "type" ]
+    },
+    {
+      "name": "apm-span",
+      "body": "index.json",
+      "types": [ "type" ]
+    }
+  ],
+  "corpora": [
+    {
+      "name": "apm-events",
```

Comment:
I think a download URL is missing here so other people can use this track as well.

Reply:
Totally, I haven't uploaded the corpora yet (mentioned that in the PR description.)

Reply:
Ok, I somehow missed that (or I've assumed that the data is uploaded meanwhile). In that case I think this is good to merge.